### DISCUSSION TITLE
Is MineColab banneable?

### DISCUSSION THREAD

**Discussion by Nangt3c** [thecoder-001/MineColab#102] [Q&A]
Could I get banned from Google Colab / Google Drive and proxy websites like playit for using this?

**Comment by N-aksif-N** [comment]
> Could I get banned from Google Colab / Google Drive and proxy websites like playit for using this?

Nah, Google Drive and Google Colab are free for everyone and also Google Colab is a product from Google, the same applies for Google Drive. We use Google Colab methods to make Minecraft server and to do stuff with it. Therefore, we have all permissions for doing this.
